imports/ui/components/music/charts.html

```diff
@@ -1,16 +1,19 @@
-<template name="charts">
-  <div class='music__charts'>
-    <h1>Music Charts</h1>
+<template name="musicCharts">
+  <div class='music__playlists'>
+    <h1>Charts</h1>
     <div>
       {{#if ready}}
-      <ul>
+      <table class="playlist-list" border="1">
       {{#each docs}}
-        <li><a href="/news/{{slug}}/">{{title}}</a></li>
+        <tr>
+          <td><img height="150px" src="{{featuredImage}}" /></td>
```

Comment:
Nitpick: img heights and table border widths should be declared in CSS rather than the HTML